extensions/amp-story/1.0/amp-story-consent.js

```diff
@@ -283,10 +286,18 @@ export class AmpStoryConsent extends AMP.BaseElement {
    * @private
    */
   onClick_(event) {
-    if (event.target && event.target.hasAttribute('on')) {
+    if (!event.target) {
+      return;
+    }
+    if (event.target.hasAttribute('on')) {
       const targetEl = dev().assertElement(event.target);
       this.actions_.trigger(targetEl, 'tap', event, ActionTrust.HIGH);
     }
+    const anchorClicked = closest(event.target, (e) => matches(e, 'a[href]'));
```

Comment:
Nit: stop at shadow root

Reply:
Closest already stops automatically at shadow root, no need to add the extra condition